tests/fixtures/covid-19 historical cases by country.csv.CSV

```diff
@@ -0,0 +1,4655 @@
+OBJECTID,ADM0_NAME,DateOfDataEntry,cum_conf,NewCase,CENTER_LON,CENTER_LAT,ADM0_VIZ_NAME,GUID,Short_Name_ZH,Short_Name_FR,Short_Name_ES,Short_Name_RU,Short_Name_AR
```

Comment:
Are we sure that it's okay to have this data publicly on github? Of course it's already public on HDX so I assume it's fine, but here it's a bit without context, so I just wanted to check.

Reply:
I think it should be ok - it is under tests/fixtures, so I think people will realise it is test data